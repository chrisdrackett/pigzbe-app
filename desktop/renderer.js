import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
