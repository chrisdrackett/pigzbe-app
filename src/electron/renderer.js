import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
