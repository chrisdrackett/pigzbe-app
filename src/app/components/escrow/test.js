import React from 'react';
import renderer from 'react-test-renderer';
import {EscrowComponent} from './';

const props = {
    dispatch: () => {},
    escrow: {
        destinationPublicKey: 'GCEMHENLZNFWDODIVJJ3L65KA6Q5I6RA6CQJKEY5VBN3KZU6MSPXR7WM',
        escrowPublicKey: 'GAHOSKJQ2UDJJ7IWRI3CMHOAQOHS435UYCP6Z3O3QU525C7DIMCGMCYW',
        transactions: [
            {
                amount: '625000',
                date: '1525278258',
                xdr: 'AAAAAA7pKTDVBpT9Foo2Jh3Ag48ub7TAn+zt24U7rovjQwRmAAAAZACFl0YAAAAGAAAAAQAAAABa6eYyAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAIjDkavLS2G4aKpTtfuqB6HUeiDwoJUTHahbtWaeZJ94AAAAAVdMTwAAAAAAN5Elfl1kQx1zgHE9fuRFc+Jxrn1+uK7y7B9BS/F6R+gAAAWvMQekAAAAAAAAAAAA',
                hash: 'a97c5c731b3de9257534025c54f9c1d12905486b16f01f208a3dab4417c17ff7',
                seq: '37602498806022150'
            },
            {
                amount: '625000',
                date: '1525278264',
                xdr: 'AAAAAA7pKTDVBpT9Foo2Jh3Ag48ub7TAn+zt24U7rovjQwRmAAAAZACFl0YAAAAHAAAAAQAAAABa6eY4AAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAIjDkavLS2G4aKpTtfuqB6HUeiDwoJUTHahbtWaeZJ94AAAAAVdMTwAAAAAAN5Elfl1kQx1zgHE9fuRFc+Jxrn1+uK7y7B9BS/F6R+gAAAWvMQekAAAAAAAAAAAA',
                hash: '5c374d271b3b104b254d981016e5bcbefcece0d0ac42c6ba133dbd855c091041',
                seq: '37602498806022151'
            },
            {
                amount: '625000',
                date: '1525278270',
                xdr: 'AAAAAA7pKTDVBpT9Foo2Jh3Ag48ub7TAn+zt24U7rovjQwRmAAAAZACFl0YAAAAIAAAAAQAAAABa6eY+AAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAIjDkavLS2G4aKpTtfuqB6HUeiDwoJUTHahbtWaeZJ94AAAAAVdMTwAAAAAAN5Elfl1kQx1zgHE9fuRFc+Jxrn1+uK7y7B9BS/F6R+gAAAWvMQekAAAAAAAAAAAA',
                hash: 'adeaa44b6a3c5eb939fa093972b8e4484bcdd104d92cd53fda375b787fcd7cb1',
                seq: '37602498806022152'
            },
            {
                amount: '625000',
                date: '1525278276',
                xdr: 'AAAAAA7pKTDVBpT9Foo2Jh3Ag48ub7TAn+zt24U7rovjQwRmAAAAZACFl0YAAAAJAAAAAQAAAABa6eZEAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAIjDkavLS2G4aKpTtfuqB6HUeiDwoJUTHahbtWaeZJ94AAAAAVdMTwAAAAAAN5Elfl1kQx1zgHE9fuRFc+Jxrn1+uK7y7B9BS/F6R+gAAAWvMQekAAAAAAAAAAAA',
                hash: '76bfcccad39648394f4818401a78875f290ce1c81e90f842fb77e8da4cef3b14',
                seq: '37602498806022153'
            }
        ]
    }
};

describe('Escrow', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<EscrowComponent {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
