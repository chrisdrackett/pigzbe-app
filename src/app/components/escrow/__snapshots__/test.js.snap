// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Escrow renders correctly 1`] = `
Array [
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgb(0, 50, 120)",
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/app/components/logo/images/pigzbe_logo_inapp.png",
        }
      }
      style={
        Object {
          "height": 25,
          "marginBottom": 10,
          "marginTop": 34,
          "width": 65,
        }
      }
    />
  </View>,
  <RCTScrollView
    contentContainerStyle={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "rgb(0, 50, 120)",
        "position": "relative",
      }
    }
    style={
      Object {
        "backgroundColor": "rgb(0, 50, 120)",
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "rgb(0, 50, 120)",
            "flex": 1,
            "justifyContent": "flex-start",
            "paddingLeft": "9.375%",
            "paddingRight": "9.375%",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "rgb(255, 255, 255)",
              "fontFamily": "Poppins",
              "fontSize": 16,
              "fontWeight": "bold",
              "marginBottom": 0,
              "marginTop": 10,
              "textAlign": "center",
            }
          }
        >
          Escrow
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "rgb(255, 255, 255)",
              "fontFamily": "Poppins",
              "fontSize": 16,
              "fontWeight": "bold",
              "marginBottom": 0,
              "marginTop": 10,
              "textAlign": "center",
            }
          }
        >
          transactions: 
          4
        </Text>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignSelf": "stretch",
              "backgroundColor": "rgb(253, 223, 6)",
              "borderColor": "rgb(253, 223, 6)",
              "borderRadius": 22.5,
              "borderWidth": 1,
              "marginBottom": 10,
              "opacity": 1,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "alignSelf": "stretch",
                "color": "rgb(0, 50, 120)",
                "fontFamily": "Poppins",
                "fontSize": 16,
                "fontWeight": "bold",
                "height": 45,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 10,
                "textAlign": "center",
              }
            }
          >
            Back
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>,
]
`;
